server/config/config_test.go

```diff
@@ -2,11 +2,12 @@ package config
 
 import (
 	"bytes"
-	"github.com/spf13/viper"
-	"github.com/stretchr/testify/assert"
 	"path/filepath"
 	"testing"
 
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
```

Comment:
Not really understanding the import sorting here. Why did it create a new import block? I think `viper`, `assert` and `require` should form only one import block.

Reply:
ah!

I can tell you what it's doing here, maybe I can explain after all.  It's separating the stdlib and external imports for readability.